packages/react/src/QuestionnaireForm/QuestionnaireFormItem/QuestionnaireFormItem.tsx

```diff
@@ -295,7 +295,12 @@ function QuestionnaireChoiceDropDownInput(props: QuestionnaireChoiceInputProps):
 function QuestionnaireChoiceSetInput(props: QuestionnaireChoiceInputProps): JSX.Element {
   const { name, item, initial, onChangeAnswer, allResponses } = props;
   if (item.answerValueSet) {
-    return <ValueSetAutocomplete binding={item.answerValueSet} onChange={onChangeAnswer} />;
+    return (
+      <ValueSetAutocomplete
```

Comment:
Is there a reason we're using `<ValueSetAutocomplete>` here rather than `<CodingInput>`?